### DISCUSSION TITLE
removing/adding objects

### DISCUSSION THREAD

**Discussion by edwagofwv** [sn4k3/UVtools#434] [Q&A] [answered]
I am very new to 3D resin printing and my first printer is the IBEE which has taken me over 6 months working
with the UNIZ people to finally get it running and now I want to try to use it and learn.
Anyone familiar with the UNIZ test dice knows there are three of them in the file.
How can I take one out to use alone and enlarge it?
How can I take an object in a file and duplicate it or add another item to the same file.
Some files print more than one item at a time and I want to know how to do the "editing"

**Comment by sn4k3** [comment, marked as answer]
> How can I take one out to use alone and enlarge it?

You only want one dice right?

1. Hold shift and drag the mouse to select all 1 dice area
2. Go to Tools - Pixel Arithmetic
3. Operator: Keep region
4. Apply

About enlarge you can't because no way for UVtools enlarge in Z, just possible in XY
But with one dice you can export it to STL using UVtools - Actions - Export to mesh in the best quaility and reslice, but they dont give the original stl of the dice?

> How can I take an object in a file and duplicate it

Tools - Pattern

>  or add another item to the same file.

1. Actions - Import layers
2. Select the file to import (It must be another sliced file)
